conf/imperial.config

```diff
@@ -31,56 +31,64 @@ profiles {
 
             // Process-specific resource requirements
             withLabel:process_single {
-                cpus   = 1
+                cpus   = 2
```

Comment:
this process label is often used for programs which can't be multi-threaded. So you will always have an idle core running it with this setup.

Reply:
ok, changing it back to 1.

Reply:
Thanks a lot for reviewing.